src/components/icon-button/icon-button.scss

```diff
@@ -17,7 +17,9 @@ $height-of-limel-button: functions.pxToRem(36);
 }
```

Comment:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/154725/212053109-11c0c1a8-8ce2-4df9-8990-870719f5194b.png">

There are some minor differences with limel-icon-button in this header example.

(I don't know why we've decided to set these weird colours on the _first_ example for limel-header? 😄 That's nothing new though, so not relevant for this PR.)